boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -83,18 +83,44 @@
 };
 
 
-&lpi2c1{
+&lpi2c1 {
 	status = "disabled";
 	clock-frequency = <I2C_BITRATE_FAST>;
 	pinctrl-0 = <&i2c1_default>;
 	pinctrl-names = "default";
 };
 
-&lpi2c2{
-	status = "disabled";
+&lpi2c2 {
+	status = "okay";
```

Comment:
that would lead to ci build failure...

Reply:
okay, but is there any test case for adp5585 gpio?

Reply:
I can see gpio tests are built successfully during ci, they should have compiled the adp5585 driver without issue. But because there's no test gpio configured in the dt overlay, there's no way we could run the test against real hardware. All the test does is to test the built-in rgpio.

So the question is, what is the typical approach to add test to these gpio expanders?